### ISSUE TITLE
Clarification on hyper-parameter search and validation set

### ISSUE THREAD

**Issue by twoertwein** [sunlightsgy/MEmoR#2]
Could you please provide more details about how hyper-parameters were validated?

Your README.txt contains the following section:
```
pretrained/
- The results reported in the paper is on the setting of train_test_9.pth and train_test_14.pth
- train_test_9.pth: Training on the training set, valid on the test set, primary emotions.
- train_test_14.pth: Training on the training set, valid on the test set, fine-grained emotions.
- train_val_test_9.pth: Training on the training set, valid on the randomly split valid set, testing on testing set, fine-grained emotions.
- train_val_test_14.pth: Training on the training set, valid on the randomly split valid set, testing on testing set, fine-grained emotions
```
Am I interpreting it correctly that the results reported in the paper are based on models that were validated on the test set?
